great_ape_safe/ape_api/rari.py

```diff
@@ -17,13 +17,21 @@ def ftoken_is_paused(self, ftoken_addr):
 
 
     def ftoken_pause(self, ftoken_addr):
+        ftoken = interface.IFToken(ftoken_addr, owner=self.safe.account)
         self.unitroller._setBorrowPaused(ftoken_addr, True)
+        if ftoken.reserveFactorMantissa() != 0:
+            ftoken._setReserveFactor(0)
         assert self.ftoken_is_paused(ftoken_addr)
+        assert ftoken.reserveFactor() == 0
 
 
-    def ftoken_unpause(self, ftoken_addr):
+    def ftoken_unpause(self, ftoken_addr, rf=0.15):
```

Comment:
isnt a default `rf` of `.15` very low? looks to me like `.65` or even `.8` would make more sense. what is the default in that vercel ui app?

Reply:
.15 rf is already on the higher end, would it makes sense to have something around ~0.05 as default instead

most pools which are more attractive has much lower than 15%, even some 0%, but i do not think so we need to go that down, as still is a protective measure

Reply:
ok i see we use .15 as a default for all ftokens so makes sense